python/cudf_polars/cudf_polars/typing/__init__.py

```diff
@@ -78,6 +79,10 @@
 
 CSECache: TypeAlias = MutableMapping[int, tuple["DataFrame", int]]
 
+ClosedInterval: TypeAlias = Literal["left", "right", "both", "none"]
+
+Duration: TypeAlias = tuple[int, int, int, int, bool, bool]
```

Comment:
I'm not usually a fan of [newtypes](https://docs.python.org/3/library/typing.html#newtype) for every little thing, but maybe for this signature it would be worthwhile? I'll defer to you :)